### PR TITLE
Fix/view species info refactor

### PR DESCRIPTION
**Pull request by chuckie512**
Cleaner, DRYer code

### CODE REVIEW

File: GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java [hunk 1]
```diff
@@ -53,54 +52,67 @@ protected void onCreate(Bundle savedInstanceState) {
...
         //set prune date TextView
-        TextView pruneDate = (TextView)findViewById(R.id.species_prune_date);
         String pruneDateString = "<b>Prune Date:</b><br />"+garden.getPruneDate(speciesName);
-        pruneDate.setText(Html.fromHtml(pruneDateString));
+        setText(R.id.species_prune_date,pruneDateString);
 
         //set size TextView
-        TextView size = (TextView)findViewById(R.id.species_size);
         String sizeString = "<b>Size:</b><br />"  + garden.getSize(speciesName);
-        size.setText(Html.fromHtml(sizeString));
```
Comment (jspodnar) [inline GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java:89]: Really nice slim-down. You saved quite a few lines here, while making it easier to read.

File: GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java [hunk 2]
```diff
@@ -53,54 +52,67 @@ protected void onCreate(Bundle savedInstanceState) {
             return;  //stop trying, nothing was received.
         }
 
+
+        updateTextViews();
```
Comment (jspodnar) [inline GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java]: I would like commenting here. I know the function is right below it, but I want to know what it does before it's called.

File: GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java [hunk 3]
```diff
@@ -53,54 +52,67 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        TextView plantDate = (TextView)findViewById(R.id.species_plant_date);
         String plantDateString = "<b>Plant Date:</b><br />"+garden.getPlantDate(speciesName);
-        plantDate.setText(Html.fromHtml(plantDateString));
+        setText(R.id.species_plant_date,plantDateString);
 
         //set prune date TextView
-        TextView pruneDate = (TextView)findViewById(R.id.species_prune_date);
         String pruneDateString = "<b>Prune Date:</b><br />"+garden.getPruneDate(speciesName);
-        pruneDate.setText(Html.fromHtml(pruneDateString));
+        setText(R.id.species_prune_date,pruneDateString);
```
Comment (jspodnar) [inline GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java:92]: So I've been doing some work into populating our plant database. After reviewing pruning, a date isn't really that important. What's important is a description of HOW to prune. I think we need to convert this into a String, and drop all the calendar events related to it. Pruning is often ongoing.

File: GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java [hunk 4]
```diff
@@ -53,54 +52,67 @@ protected void onCreate(Bundle savedInstanceState) {
...
                 //+ "</font>"
                 ;
-        
-        color.setText(Html.fromHtml(colorString));
-
+        setText(R.id.species_color,colorString);
 
+    }
 
+    /**
```
Comment (jspodnar) [inline GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java:108]: Nice work.

File: GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java (hunk 2 above)
Comment (chuckie512) [inline GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java]: added comment

File: GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java (hunk 3 above)
Comment (chuckie512) [inline GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java:92]: out of scope for this PR

**Pull request merged**